packs/feat-effects/effect-emblazon-armament-shield.json

```diff
@@ -0,0 +1,42 @@
+{
+    "_id": "o04gwnrVtzWyIEs8",
+    "img": "systems/pf2e/icons/spells/dancing-shield.webp",
+    "name": "Effect: Emblazon Armament (Shield)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Emblazon Armament] when applied to a shield.</p>\n<ul>\n<li><strong>Shield</strong> The shield gains a +1 status bonus to its Hardness.</li>\n</ul>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
```

Comment:
Use ItemAlteration instead. Take a look at sapling shield for an example.

Reply:
Updated effect to use choice set to select shield, then ItemAlteration to upgrade hardness